src/loc.rs

```diff
@@ -27,7 +27,7 @@ pub struct LocDiff {
 impl LocDiff {
     /// The getter method for the field `file` of the corresponding struct.
     ///
-    /// Since the struct `Loc` defines its field `file` to be private, this
+    /// Since the struct `LocDiff` defines its field `file` to be private, this
```

Comment:
Same here. This should have been squashed with the commit, that introduced that renaming..